pkg/config/config.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const authPathFmtString = "auth/%s/login"
 
-//Config represents an application configurations
+// Config represents an application configurations
```

Comment:
Suggestion: s/configurations/configuration/

Reply:
Done! (I'll also likely be doing more of these types of changes as part of future any PRs.)